src/components/SotaChart.js

```diff
@@ -28,77 +30,37 @@ class SotaChart extends React.Component {
       chart: null,
       task: {},
       isLog: props.isLog ? 1 : 0,
-      chartKey: '',
-      chartKeyInt: 0,
+      logBase: Math.log(10),
+      log: Math.log10,
+      subset: '',
+      label: 'method',
+      metricNames: {},
+      chartKey: null,
```

Comment:
Is there a reason it went from `''` to `null`? I'm honestly not sure if one is preferable over the other here for this MR. Is there a reason you made this change?